components/ma-map-edit.vue

```diff
@@ -138,7 +138,7 @@
                         </p>
                     </div>
                 </div>
-                <div class="divider">
+                <!-- <div class="divider">
```

Comment:
@mmnoo seems this didn't work

Reply:
Could this be a CDN caching thing? Is it worth waiting?